### DISCUSSION TITLE
Glyph doesn't display

### DISCUSSION THREAD

**Discussion by CarsonBuckley** [ohmyzsh/ohmyzsh#11382] [Help] [answered]
![Screen Shot 2022-12-09 at 1 08 35 PM](https://user-images.githubusercontent.com/45154466/206788384-0bfe5a9f-1500-474f-8f95-ddb074375223.png)

I'm using the 'robbyrussell' theme, but the 'X' symbol doesn't appear after my branch. Yes, I've installed and selected multiple different nerd fonts to try to get it to display, but can't figure it out.

**Comment by CarsonBuckley** [comment]
<img width="897" alt="Screen Shot 2022-12-09 at 1 15 57 PM" src="https://user-images.githubusercontent.com/45154466/206788913-2a1a6763-8372-454c-a33b-eb416d21d10f.png">

Here is the 'X' glyph and where it should be displayed for reference

**Comment by carlosala** [comment, marked as answer]
That only appears if the repo is dirty. Which is the result of your `git status`?
